src/bitrot.py

```diff
@@ -180,12 +179,13 @@ def run(self):
         errors = []
         current_size = 0
         missing_paths = self.select_all_paths(cur)
-        paths, total_size = list_existing_paths(
+        hashes_dict_db = self.select_all_hashes(cur)
+        paths_fs, total_size = list_existing_paths(
```

Comment:
Can you keep the `paths` name as it was before? I don't see the value of changing it.